src/tools/dev/scripts/bv_support/bv_adios2.sh

```diff
@@ -182,7 +182,7 @@ function build_adios2
             fi
 
             # Change all references from adios2 to adios2_mpi.
-            find . -name "CMakeLists.txt" -exec sed -i "s/adios2/adios2_mpi/g" {} \;
+            find . -name "CMakeLists.txt" -exec ${SED_CMD} "s/adios2/adios2_mpi/g" {} \;
```

Comment:
where is `SED_CMD` defined? Is the expectation this is defined in the environment or on the command line via something like `env SED_CMD=/foo/gar/sed ./build_visit`? There appear to be many other instances in the use of `sed` in `build_visit` such as, for example, https://github.com/visit-dav/visit/blob/80c2e3f54eff2adde52c33cb7c3c884031c04053/src/tools/dev/scripts/bv_support/helper_funcs.sh#L910

and am wondering if you think it might requires more formal, full-fledged support?

Reply:
SED_CMD is defined a few lines up in the file. The purpose it to capture the slight differences when calling it on macOS and other systems.